app/routes/index.js

```diff
@@ -0,0 +1,7 @@
+const healthCheckRoutes = require('./healthCheck');
+const jsonPlaceHolderRoutes = require('./jsonPlaceHolder');
+
+exports.init = app => {
+  app.use('/health', healthCheckRoutes);
+  app.use('/json-place-holder', jsonPlaceHolderRoutes);
```

Comment:
Fixed it! :)

Reply:
Here the idea is to avoid showing the users or client what are we consuming in our services and when you put `/json-place-holder` you are doing the opposite. You can rewrite the routes file but try to keep the standard.

Reply:
I try to understand your point.

You says that the user that is consuming an endpoint shouldn't know the parent service that is being consumed from.

Lets imagine an example:

We have a microservice `A` and microservice `B`.

`A` has an endpoint `/image` and `B` also has an endpoint `/image` in his context.

If `A` requires to call `/image` of `B`, how should be declared it? I'd declare as `/b/image`.

If I call it as `/image`, it will matching with `/image` of `A`.

I have the following questions: 

- How should be declared `/image` of `B` in `A`?
- Why users shouldn't know it? Because in all of previous projects I've worked, we always declared routes like this and as far as I understand this doesn't represent a vulnerability bridge (I'm not try say that it's correct necessarily).
- Can you show me how would you declare it, please?

Note: I don't want you to think that I want to have the reason or confronting to you (NEVER) ,I never pretend it, just want to know and learn the best practices in everything, it's my unique interest :).